common/src/main/java/com/wynntils/services/destination/DestinationService.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.destination;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Managers;
+import com.wynntils.core.components.Service;
+import com.wynntils.core.net.Download;
+import com.wynntils.core.net.UrlId;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DestinationService extends Service {
+    // Map<location, abbreviation>
+    private Map<String, String> destinations;
+
+    public DestinationService() {
+        super(List.of());
+
+        loadDestinations();
+    }
+
+    @Override
+    public void reloadData() {
+        loadDestinations();
+    }
+
+    /**
+     * Get the abbreviation of a location
+     * @param location Full location name of the destination (eg. "Ragni")
+     * @return Abbreviation of the location (eg. "Ra") or null if not found
+     */
+    public String getAbbreviation(String location) {
+        if (!destinations.containsKey(location)) {
+            return null;
+        }
+        return destinations.get(location);
```

Comment:
I don't think the containsKey check is needed as it should return null if the key isn't found

Reply:
Doesn't it cause npe?

Reply:
Though if you do move the fallback here, then it would be done inside the if

Reply:
> Doesn't it cause npe?

I believe it just returns null, might need to double check